openff/cli/tests/test_cli_functions.py

```diff
@@ -0,0 +1,330 @@
+import pathlib
+import tempfile
+
+import numpy as np
+import pytest
+from openforcefield.topology import Molecule
+from openforcefield.typing.engines.smirnoff.forcefield import get_available_force_fields
+from openforcefield.utils import (
+    OPENEYE_AVAILABLE,
+    RDKIT_AVAILABLE,
+    OpenEyeToolkitWrapper,
+    RDKitToolkitWrapper,
+    temporary_cd,
+)
+
+from openff.cli.check_versions import get_versions
+from openff.cli.generate_conformers import (
+    generate_conformers,
+    make_registry,
+    write_mols,
+)
+from openff.cli.tests.utils import get_data_file_path
+
+# TODO: Run all tests in a safe temporary directory
+
+
+class TestCheckVersions:
+    def test_basic_output(self):
+        out = get_versions()
+        from openforcefield import __version__ as toolkit_version
+
+        from openff.cli import __version__ as cli_version
+
+        assert "OpenFF Toolkit" in out
+        assert "OpenFF CLI" in out
+        assert toolkit_version in out
+        assert cli_version in out
+
+
+# TODO: Update skipifs after #615
```

Comment:
(not blocking) [#615](https://github.com/openforcefield/openforcefield/pull/615) is now merged, though at a quick glance I don't know how it affects this.

Reply:
It was more or less cosmetic